bin/asl-say

```diff
@@ -43,60 +45,72 @@ function build_and_say(){
 }
 
 function get_time(){
-	
+
 	# calculate the hour
-	NOW_H=$(date +%I | sed -e 's/0//g')
-
-	# calculate the minute audio	
-	NOW_M=$(date +%M | sed -e s/'^0//')
-	NOW_FM=$(( ${NOW_M} / 10 ))
-	case ${NOW_FM} in
-		0)	NOW_FM=0 ;;
-		1)	NOW_FM=$(date +%M) ;;
-		*)	NOW_FM="${NOW_FM}0" ;;
-	esac
-	NOW_SM=$(( ${NOW_M} % 10 ))
-	if [ $NOW_FM -eq 0 ] && [ $NOW_SM -eq 0 ]; then
-		$NOW_FM=oclock
-		$NOW_SM=""
+	HOUR=$($DATE_CMD +%I | sed -e 's/0//g')
```

Comment:
This always strips out `0` so it is turning `10` PM into `1` PM on my system, which is not correct.